libfuse/fs.go

```diff
@@ -260,6 +281,25 @@ func (f *FS) WithContext(ctx context.Context) context.Context {
 	return ctx
 }
 
+func (f *FS) maybeStartTrace(
+	ctx context.Context, family, title string) context.Context {
+	// TODO: Add options to enable/disable tracing, or adjust
+	// trace detail.
+	tr := trace.New(family, title)
```

Comment:
Is this expensive to have on by default?  Does it do anything when the http server is disabled?